src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -875,72 +881,78 @@ await using (var context = InMemoryApplicationDbContext(contextId))
         }
 
         [Fact]
-        public async void GetReleaseSummaryAsync()
+        public async void GetRelease()
         {
-            var releaseId = new Guid("5cf345d4-7f7b-425c-8267-de785cfc040b");
-
-            var adhocReleaseType = new ReleaseType
+            var adHocReleaseType = new ReleaseType
             {
-                Id = new Guid("19b024dc-339c-4e2c-b2ca-b55e5c509ad2"),
                 Title = "Ad Hoc"
             };
 
-            var publishScheduled = new DateTime(2020, 6, 29, 0, 0, 0).AsStartOfDayUtcForTimeZone();
+            var releaseId = Guid.NewGuid();
             var nextReleaseDate = new PartialDate {Day = "1", Month = "1", Year = "2040"};
-            const string releaseName = "2035";
-            const TimeIdentifier timePeriodCoverage = TimeIdentifier.January;
+
+            var release = new Release
+            {
+                Id = releaseId,
+                Type = adHocReleaseType,
+                TimePeriodCoverage = TimeIdentifier.January,
+                PublishScheduled = DateTime.Parse("2020-06-29T00:00:00.00Z", styles: DateTimeStyles.AdjustToUniversal),
```

Comment:
I don't think anything is changing on this PR to do with the way the Pre-release access window is changing so I'm just wondering why you are dropping the `.AsStartOfDayUtcForTimeZone()` on the publish scheduled date?

Reply:
Basically I didn't want the test to be dependent on a util function as it's not really what we're interested in. Also using `DateTime.Parse` sets the date time's kind to local, meaning that `AsStartOfDayUtcForTimeZone` ends up throwing an error anyway (it needs it to be UTC).

Reply:
Ok, that makes sense 👍 .